File: C++/program147.cpp
#include<iostream>
using namespace std;

bool CheckCapital(char ch)
{
    bool bFlag = false;

    if((ch >= 'A') && (ch <= 'Z'))
    {
        bFlag = true;
    }

    return bFlag;
}

int main()
{
    char cValue = '\0';
    bool bRet = false;

    cout<<"Enter charcter"<<'\n';
    cin>>cValue;

    bRet = CheckCapital(cValue);

    if(bRet == true)
    {
        cout<<"charcter is capital"<<'\n';
    }
    else
    {
        cout<<"charcter is not capital"<<'\n';
    }
    return 0;
}

File: program147.java
import java.util.*;

class CharcterProgram
{
    public boolean CheckCapital(char ch)
    {
        if((ch >= 65) && (ch <= 90))
        {
            return true;
        }
        else
        {
            return false;
        }

    }
}

class program147
{
    public static void main(String A[])
    {
        char cValue =' ';
        boolean bRet = false;
        
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter charcter :");
        cValue = sobj.next().charAt(0);

        CharcterProgram cobj = new CharcterProgram();
        bRet = cobj.CheckCapital(cValue);

        if(bRet == true)
        {
            System.out.println(cValue+" is capital");
        }
        else
        {
            System.out.println(cValue + " is not capital");
        }

     
         

    }
}